swift_browser_ui_frontend/src/components/EditTagsModal.vue

```diff
@@ -123,11 +151,29 @@ export default {
         if (this.$route.name !== "SharedObjects") {
           await this.$store.state.db.objects
             .where(":id").equals(this.object.id)
-            .modify({tags: this.object.tags});
+            .modify({tags: this.tags});
         }
+        // FIXME: Looks like it's missing to save SharedObject tags
```

Comment:
is this a new issue ? not sure how to interpret this, more details would be nice

Reply:
Yes, it might be a bug. I should have removed that one before pushing ^^. Shared objects doesn't use IndexedDB, and without saving it to memory, the UI might not update. But I didn't look into it.

Reply:
ok, then as long as it is created as bug, i am ok.